app/controllers/codeland_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CodelandController < ApplicationController
+  # No authorization required for viewing Codeland
+
+  def show
+    return if Flipper[:codeland].enabled?(current_user)
```

Comment:
Mental note to add a little wrapper on these so that we don't sprinkle "Flipper" everywhere when we start using it

Reply:
Is your objection to the name `Flipper` in code? We definitely wrap that in a more neutral name.

Reply:
maybe we could wrap it with `FeatureFlags`?